_studio/mfx_lib/Android.mk

```diff
@@ -334,7 +334,7 @@ LOCAL_CFLAGS_64 := $(MFX_CFLAGS_INTERNAL_64)
 
 LOCAL_LDFLAGS := $(MFX_LOCAL_LDFLAGS_HW)
 
-LOCAL_HEADER_LIBRARIES := libmfx_headers
+LOCAL_HEADER_LIBRARIES := libmfx_headers libva_headers
```

Comment:
Hi Jeevaka, could you please share where libva_headers is declared in MediaSDK? I don't see it in this change. By the way, why do we need this change here? Is it for the compiling issue on Android 15?

Thanks,
Tianmi